src/modules/swap/ModuleSwap.vue

```diff
@@ -20,21 +20,22 @@
               From / Amount to Swap / To / Amount to Recieve
             =====================================================================================
             -->
-          <v-row class="align-center justify-space-between">
-            <v-col cols="12">
-              <v-skeleton-loader v-if="isLoading" type="text" width="375px" />
-              <div v-else class="available-balance text-right">
-                {{ balanceInETH }}
-              </div>
-            </v-col>
+          <v-row class="align-center justify-space-between mt-4">
             <v-col cols="12" sm="5" class="pb-0 pb-sm-3">
-              <mew-select
-                :value="fromTokenType"
-                label="From"
-                :items="actualFromTokens"
-                :is-swap="true"
-                :loading="isLoading"
-                @input="setFromToken" />
+              <div class="position--relative">
+                <wallet-top-right-text
```

Comment:
- mew-component--top-right-text class makes the element `right: 8px`; of right to the component, In the design the element is positioned as` right: 0px`
- change color to green

Reply:
fixed